lib/modules/manager/helmfile/__fixtures__/helmfile-two.lock

```diff
@@ -0,0 +1,10 @@
+version: 0.150.0
```

Comment:
please inline those fixtures and use `codeBlock`  string template for format help.